CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+0.0.5
+- Support queries for and by dataset
+- Support queries for all entities by identifier
+- Examples of query by cluster
+- Example of query for cell by protein
+- Reworking client-side parameter validation
+- Integrating hubmap_cell_id_gen_py into examples
+
```

Comment:
Cool: I still have 0.0.4 marked as "in progress": it hasn't been released yet, so the additions go there, until we tag it.

Reply:
Oh, right.  Will do.